tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- assert:
+    that: "'{{ LOAD_BALANCER_SERVER_IP | ipaddr }}' == '{{ LOAD_BALANCER_SERVER_IP }}'"
```

Comment:
@pomegranited The `ipaddr` filter normalises the IP address, so the output is different from the input in certain cases.  The filter returns `False` if the input isn't an IP address in the first place, so the `== '{{ LOAD_BALANCER_SERVER_IP }}'` part should be removed.